cni/pkg/iptables/iptables.go

```diff
@@ -141,14 +141,14 @@ func (cfg *IptablesConfigurator) executeDeleteCommands() error {
 	deleteCmds := [][]string{
 		{"-t", iptablesconstants.MANGLE, "-D", iptablesconstants.PREROUTING, "-j", ChainInpodPrerouting},
 		{"-t", iptablesconstants.MANGLE, "-D", iptablesconstants.OUTPUT, "-j", ChainInpodOutput},
-		{"-t", iptablesconstants.NAT, "-D", iptablesconstants.PREROUTING, "-j", ChainInpodPrerouting},
 		{"-t", iptablesconstants.NAT, "-D", iptablesconstants.OUTPUT, "-j", ChainInpodOutput},
-		{"-t", iptablesconstants.RAW, "-D", iptablesconstants.PREROUTING, "-j", ChainInpodPrerouting},
-		{"-t", iptablesconstants.RAW, "-D", iptablesconstants.OUTPUT, "-j", ChainInpodOutput},
 	}
 
-	// these sometimes fail due to "Device or resource busy"
+	// these sometimes fail due to "Device or resource busy" or because they are optional given the iptables cfg
 	optionalDeleteCmds := [][]string{
```

Comment:
Thoughts on considering all delete cmds optional?

Reply:
I see no problem with having everything optional but I would like to have @bleggett opinion on this as I think this affects agent-server as well.

Reply:
All deletes optional is OK I believe - we do deletes on startup as a cleanup mechanism in the host netns, and knowing if those failed 

- because the chains aren't there (in which case, can swallow the error)
- because `iptables` and/or the kernel have a problem (in which case, we shouldn't)

would be preferred.

We will fail later on most likely anyway for the second case tho, so I think it's fine.